chore: add `pem_cache` cli (#1085)
emqtt/emq-docs-cn: en_US/advanced/cli.md

  ok
  ```

+ acl del `_all` `<Topic>`

  Delete $all acl

  ```bash
  ./bin/emqx_ctl acl del _all Topic/C
  ok
  ```

## pem_cache command

> The `pem_cache` command is used to clear the cache of all SSL-related certificates.

Reviewer: ```suggestion
The `pem_cache` command is used to clear the cache of all PEM certificates.
```

> For example, after replacing the SSL listener's certificate file, you can use this command

Reviewer: ```suggestion
For example, after some new trusted certificates are added to `cacertfile`
or server certificate `certfile` renewal for SSL listeners, you can use this command
```

> to make the new certificate file take effect instead of restarting EMQX's SSL listener.

Reviewer: ```suggestion
to make the new certificate file take effect instead of restarting EMQX's SSL listener.

**NOTE** The updates of the cache do not affect currently established TLS connections.
```

| Command                         | Description                                 |
| ------------------------------- | ------------------------------------------- |
| `pem_cache clean              ` | Clears x509 certificate cache on all nodes  |
| `pem_cache clean node <Node>  ` | Clears x509 certificate cache on given node |

::: tip

> This command was introduced in v4.3.13. It was not supported in any version prior to this.

Reviewer: this PR should be sent to `release-4.3`

Author: Fixed. Thanks for your correction.
